<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Справочник</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>guide_icon.png</normaloff>guide_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="HT_table">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>31</y>
      <width>1251</width>
      <height>391</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Госномер</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Номер телефона</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Марка, модель</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Дата</string>
     </property>
    </column>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>1261</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QSpinBox" name="Add_HT_Size">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="loadButton">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>20</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Загрузить данные</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveButton">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>20</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Введите размер хэш-таблицы:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="size_HT_Button">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="debugButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>600</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Отладка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>640</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>519</y>
      <width>250</width>
      <height>140</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Госномер</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditGosNumber">
     <property name="geometry">
      <rect>
       <x>52</x>
       <y>60</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>520</y>
      <width>250</width>
      <height>140</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Номер телефона</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditPhoneNumber">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>60</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>520</y>
      <width>250</width>
      <height>140</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Марка,модель</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCarModel">
     <property name="geometry">
      <rect>
       <x>52</x>
       <y>60</y>
       <width>151</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>520</y>
      <width>250</width>
      <height>140</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Дата</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
